client/components/gridicon/docs/example.jsx

```diff
@@ -19,7 +19,8 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<div>
+			<div className="design-assets__group">
```

Comment:
yes @jasmussen, at least for now. One idea is, in a near future, use a component to wrap the example and implement some utilities.